src/authenticated/onboarding/OnboardingPage.tsx

```diff
@@ -25,56 +28,47 @@ function formatMemberForMutation(updatedUser: MemberProfileFragmentFragment) {
   };
 }
 
-const MemberEditPage: React.SFC<MemberEditPageProps> = ({ history }) => {
+const MemberEditPage: React.FC<MemberEditPageProps> = ({ history }) => {
   const authContext = useContext(AuthenticationContext);
+  const updateMemberMutation = useMutation<UpdateUserMutation>(
+    UpdateUserDocument,
+    {
+      refetchQueries: ['editableUsersList']
+    }
+  );
+  const { data } = useCustomQuery<GetUserQuery>(GetUserDocument, {
+    variables: { id: authContext.authData.userId }
+  });
+  if (!data || !data.user) return null;
```

Comment:
I see this pattern a lot in your code. It is probably always better to inform the user that there was no data found rather than returning `null`.

Reply:
This actually should never happen. This is just to appease the TypeScript type checker because I don't know enough advanced TypeScript to specify that the output at this point should never be undefined.